### DISCUSSION TITLE
Description field to be manadatory input

### DISCUSSION THREAD

**Discussion by divemasterza** [kimai/kimai#4125] [General]
Hi,

Is there any way to make the description a mandatory input? Could not find it in the settings.
Perhaps something that can be achieved with local.yaml?

**Comment by kevinpapst** [comment]
The search field can be used to type in stuff like "description", which then brings up great ideas 😁 
https://github.com/kimai/kimai/discussions/3680#discussioncomment-4333904

**Reply by raymondgeurden** [reply]
We would also welcome the option to set Description and Tags as mandatory fields. Maybe even the option to set a minimum number of characters for the Desciption to report on activities.

Is this an issue on your backlog?

**Reply by kevinpapst** [reply]
No, because the linked comment shows you how to achieve this functionality with the custom content plugin and some javascript.
